main.js

```diff
@@ -34,70 +34,36 @@ app.get('/build',function(req,res,next){
   res.render('build', context);
 });
 
-/*
-display recipes for breakfast
-*/
-app.get('/breakfast',function(req,res,next){
-  let context = {};
-  context.title = "Breakfast";
-
-  // Select all from the test_table
-  let query = "select r.name as recipeName, c.name as categoryName from recipe r inner join recipe_category rc on r.id = rc.recipe_id inner join category c on rc.category_id = c.id where rc.category_id=1";
-
-  pg.query(query, (err, result) => {
-    if(err){
-      next(err);
-      return;
-    }
-
-    context.results = result.rows;
-    res.render('breakfast', context);
-  });
-});
 
 /*
-display recipes for lunch
+display on category page based on click
 */
-app.get('/lunch',function(req,res,next){
-  let context = {};
-  context.title = "Lunch";
-
-  // Select all from the test_table
-  let query = "select r.name as recipeName, c.name as categoryName from recipe r inner join recipe_category rc on r.id = rc.recipe_id inner join category c on rc.category_id = c.id where rc.category_id=2";
 
-  pg.query(query, (err, result) => {
-    if(err){
-      next(err);
-      return;
-    }
-
-    context.results = result.rows;
-    res.render('lunch', context);
-  });
-});
-
-
-/*
-display recipes for dinner
-*/
-app.get('/dinner',function(req,res,next){
+app.get('/category/:catname',function(req,res,next){
```

Comment:
Can we simply use `/category/:category`, instead of `/category/:catname`?

Reply:
I updated this for clarity.